master/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Starting off with the Jenkins base Image
+FROM jenkinsci/blueocean:latest
+ 
+# Installing the plugins we need using the in-built install-plugins.sh script
+RUN /usr/local/bin/install-plugins.sh \ 
+    git git-parameter matrix-auth parameterized-trigger workflow-aggregator ssh-agent ssh-slaves \ 
+    docker-workflow credentials-binding htmlpublisher xcode-plugin
+
+# Setting up environment variables for Jenkins admin user
+ENV JENKINS_USER admin
+ENV JENKINS_PASS admin
```

Comment:
Minor improvement: to save on intermediate layers, her we can combine the ENVs into one line:

```
ENV JENKINS_USER admin \ 
    JENKINS_PASS admin
```